brave/browser-laptop
# Refactor ReleaseNotes with Aphrodite

## Test Plan
1. Test that updating using BRAVE_UPDATE_VERSION=0.8.3 env variable works correctly
2. Click "Details"
3. Check the design of the dialog

## Description
Closes #7171

Auditors: @cezaraugusto 


Test Plan:
